f8a_worker/storages/s3_manual_tagging.py

```diff
@@ -0,0 +1,28 @@
+from . import AmazonS3
+
+
+class S3ManualTagging(AmazonS3):
+    def store_user_data(self, input_json):
+        file_name = self.get_file_name(input_json)
+        data = input_json.get('data', {})
+        ecosystem = data.get('ecosystem', None)
+        if ecosystem != 'pypi':
+            file_name = ecosystem + "/" + file_name
+        self.store_dict(data, file_name)
+
+    def fetch_user_data(self, input_json):
+        user_json = {}
+        ecosystem = input_json.get('ecosystem', None)
```

Comment:
Same error as bellow - concantenating None with a string results in error. I would suggest to directly pass ecosystem as parameters as well as user.